src/devices/Ssd13xx/Ssd1309.cs

```diff
@@ -0,0 +1,227 @@
+
+        /// <summary>
+        /// Sends command to the device
+        /// </summary>
+        /// <param name="command">Command being sent</param>
+        public override void SendCommand(ISharedCommand command) => SendCommand(command);
+
+        /// <summary>
+        /// Send a command to the display controller.
+        /// </summary>
+        /// <param name="command">The command to send to the display controller.</param>
+        private void SendCommand(ICommand command)
+        {
+            Span<byte> commandBytes = command?.GetBytes();
+
+            if (commandBytes is not { Length: >0 })
+            {
+                throw new ArgumentNullException(nameof(command), "Argument is either null or there were no bytes to send.");
+            }
+
+            // TODO: Is this needed at all if data/command is controlled via GPIO instead of a control byte?
+            Span<byte> writeBuffer = SliceGenericBuffer(commandBytes.Length);
+
+            commandBytes.CopyTo(writeBuffer.Slice(0));
+
+            if (_spiDevice != null)
+            {
+                // Begin consuming SPI bus data
+                // Because the timing is not perfect, this may have adverse side-effects when using multiple SPI bus devices
+                _csGpioPin.Write(PinValue.Low);
+
+                // Enable command mode
+                _dcGpioPin.Write(PinValue.Low);
+                _spiDevice.Write(writeBuffer);
+
+                // Stop consuming SPI bus data
+                _csGpioPin.Write(PinValue.High);
+            }
+            else
+            {
+                throw new InvalidOperationException("No SPI device available or it has been disposed.");
+            }
+        }
+
+        /// <summary>
+        /// Sends data to the device
+        /// </summary>
+        /// <param name="data">Data being sent</param>
+        public override void SendData(Span<byte> data)
+        {
+            if (data.IsEmpty)
+            {
+                throw new ArgumentNullException(nameof(data));
+            }
+
+            if (_spiDevice != null)
+            {
+                // Begin consuming SPI bus data
+                // Because the timing is not perfect, this may have adverse side-effects when using multiple SPI bus devices
+                _csGpioPin.Write(PinValue.Low);
+
+                // Enable data mode
+                _dcGpioPin.Write(PinValue.High);
+                _spiDevice.Write(data);
+
+                // Stop consuming SPI bus data
+                _csGpioPin.Write(PinValue.High);
+            }
+            else
+            {
+                throw new InvalidOperationException("No SPI device available or it has been disposed.");
+            }
+        }
+
+        /// <summary>
+        /// Init 128x64
+        /// </summary>
+        protected virtual void Initialize()
+        {
+            SendCommand(new SetDisplayOff());
```

Comment:
possible to get couple of comments on the commands where there are magic numbers? It will make the maintainability a bit better

Reply:
Sure, I did not add any new commands but instead used the Ssd1306 commands with the namespace prefix removed here for less confusion. I will go through the commands and make sure they are all documented and relevant to the Ssd1309 though. For example, I don't think the Ssd1309 has a charge pump like the Ssd1306 but I need to confirm.

Reply:
I addressed this by removing the magic numbers and using the default values which have comments describing their meaning